config/configtls/configtls.go

```diff
@@ -153,6 +153,24 @@ func (r *certReloader) GetCertificate() (*tls.Certificate, error) {
 	return r.cert, nil
 }
 
+func (c TLSSetting) Validate() error {
+	minTLS, err := convertVersion(c.MinVersion, defaultMinTLSVersion)
+	if err != nil {
+		return fmt.Errorf("invalid TLS min_version: %w", err)
+	}
+
+	maxTLS, err := convertVersion(c.MaxVersion, defaultMaxTLSVersion)
```

Comment:
`convertVersion` is also used in [TLSSetting.loadTLSConfig()](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/configtls.go#L196). Both client and server implementation call it in `LoadTLSConfig()`. Isn't it a problem that the logic that validates TLS version is used in load methods and `Validate()`?

Reply:
What `Validate()` do you mean?

Reply:
Yes, I think this is fine